src/main/kotlin/com/wgslfuzz/semanticspreservingtransformations/ControlFlowWrapping.kt

```diff
@@ -0,0 +1,374 @@
+/*
```

Comment:
General comment on this file - please add more by way of comments. I didn't do a full review of all the details yet, but more comments would help me in doing so.